apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx

```diff
@@ -849,262 +927,273 @@ export const BaseInvoiceForm: React.FC<BaseInvoiceFormProps> = (props) => {
             validateTrigger={FORM_VALIDATE_TRIGGER}
             {...modalFormProps}
             children={({ handleSave, form }) => (
-                <Row gutter={OUTER_VERTICAL_GUTTER}>
-                    <Col span={24} md={colSpan} hidden={isModalForm}>
+                <>
+                    <Prompt
+                        title={SaveChangesModalTitle}
+                        form={form}
+                        handleSave={handleSave}
+                    >
+                        <Typography.Paragraph>
+                            {SaveChangesNodalMessage}
+                        </Typography.Paragraph>
+                    </Prompt>
```

Comment:
Just added `<> <Prompt ... /> {oldCode}  </>` here. 
Changes below in this file because of this.